.github/workflows/main.yml

```diff
@@ -0,0 +1,63 @@
+name: react-native-square-reader-SDK
+on: [push]
+jobs:
+  install-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+      - run: |
+          yarn && yarn lint && yarn test && cd reader-sdk-react-native-quickstart && yarn
```

Comment:
can you please break the line before cd to quickstart folder?

Reply:
Done !!